### PR TITLE
Los 39

### PR DESCRIPTION
**Pull request by brianbegygr**
Upgrade of Encompass SDK to 17.1.  Return to using Nuget for package management.  Add in documentation on how to build a Nuget package for the Encopmass SDK.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## v17.1.0.0 / 2017 Feb 08
+> This release updates the Encompass SDK to version 17.1.
```
Comment (jeffpsherman) [inline CHANGELOG.md]: Should also mention that we've added code/scripts to build your own Encompass SDK nuget

File: GuaranteedRate.SextantTests/GuaranteedRate.SextantTests.csproj [hunk 2]
```diff
@@ -35,17 +35,159 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="EllieMae.Encompass.Runtime">
-      <HintPath>C:\Program Files (x86)\Encompass\EllieMae.Encompass.Runtime.dll</HintPath>
+    <Reference Include="acPDFCreator.Net, Version=4.0.0.8, Culture=neutral, PublicKeyToken=8aedc7a81d17941d, processorArchitecture=x86">
```
Comment (jeffpsherman) [inline GuaranteedRate.SextantTests/GuaranteedRate.SextantTests.csproj]: This looks wrong - I thought the acPDFCreator was part of the stuff that we didn't want to include.
It's also not on the list of dlls to make the encompass nuget pack

Comment (brianbegygr) [inline GuaranteedRate.SextantTests/GuaranteedRate.SextantTests.csproj]: Yeah looks like the tests project has a bunch of weird stuff not from NuGet.  Fixing.

File: CHANGELOG.md (hunk 1 above)
Comment (brianbegygr) [inline CHANGELOG.md]: It is in the details, but I'll add it to the header.

**Comment by jeffpsherman** [comment]
👍

**Pull request merged**